lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -382,6 +388,7 @@ def commits_in_tag(sha, shas = Set.new)
       shas
     end
 
+    # @param [Object] indata
```

Comment:
These are super cool additions, and type descriptions in Yard can use a comma to separate each matching type like [Array, Hash] etc.